src/resources/views/crud/columns/enum.blade.php

```diff
@@ -0,0 +1,35 @@
+{{-- checkbox with loose false/null/0 checking --}}
```

Comment:
Please stop working on this. Remove it. Pedro is already working on it in https://github.com/Laravel-Backpack/CRUD/pull/4678/files/044bd47b245ce855649a9f7e6afee1f5191f05bc..8eb076ef2e0052ece79287e0f0b3dc5c55f69384